doc/doc-support/lib-function-docs.nix

```diff
@@ -22,6 +22,7 @@ with pkgs; stdenv.mkDerivation {
     docgen lists 'List manipulation functions'
     docgen debug 'Debugging functions'
     docgen options 'NixOS / nixpkgs option handling'
+    docgen path 'Path functions'
```

Comment:
Also touched by https://github.com/NixOS/nixpkgs/pull/205643, so perhaps you could review.

Reply:
Yup saw it, I might take a look